src/App.js

```diff
@@ -6,6 +6,7 @@ import EditEvent from "./components/dashboard/EditEvents";
 import Profile from "./components/profile";
 import DetailEvent from "./components/dashboard/DetailEvent";
 import SignUp from "./components/signUp/SignUp";
+import NewEvent from "./components/newEvent/NewEvent";
```

Comment:
lets place the `NewEvent` component inside events `folder` so the path should be: "./components/events/NewEvent";